server/routerlicious/packages/services-ordering-rdkafka/src/rdkafkaBase.ts

```diff
@@ -70,6 +71,20 @@ export abstract class RdkafkaBase extends EventEmitter {
 				"security.protocol": "ssl",
 				"ssl.ca.location": options?.sslCACertFilePath,
 			};
+		} else if (options?.eventHubConnString) {
```

Comment:
Is it possible that options?.sslCACertFilePath and options?.eventHubConnString are true?
If yes, then sslOptions corresponding to eventHubConnString will not be set.

Reply:
It's intentional, they are supposed to be mutual excluding